### ISSUE TITLE
Why is it not possible to specify default migration directory for cli tool?

### ISSUE THREAD

**Issue by KieronWiltshire** [typeorm/typeorm#9210]
### The Problem

When using TypeORM with frameworks like NestJS or just in general, you have to specify the full path for migration files. This can be a developer onboarding headache. "I've told you to always make the migrations inside the migrations directory."

Why is there no possible way to define this in the typeorm config?

### The Solution

Allow the ability to specify the migration directory via typeorm config (this was done in previous versions but not sure why this no longer exists)

### Relevant Database Driver(s)

| DB Type           | Relevant      |
|-------------------| ---           |
| `aurora-mysql`    | no            |
| `aurora-postgres` | no            |
| `better-sqlite3`  | no            |
| `cockroachdb`     | no            |
| `cordova`         | no            |
| `expo`            | no            |
| `mongodb`         | no            |
| `mysql`           | no            |
| `nativescript`    | no            |
| `oracle`          | no            |
| `postgres`        | no            |
| `react-native`    | no            |
| `sap`             | no            |
| `spanner`         | no            |
| `sqlite`          | no            |
| `sqlite-abstract` | no            |
| `sqljs`           | no            |
| `sqlserver`       | no            |

### Are you willing to resolve this issue by submitting a Pull Request?

 - ✖️ Yes, I have the time, and I know how to start.
 - ✖️ Yes, I have the time, but I don't know how to start. I would need guidance.
 - ✖️ No, I don’t have the time, but I can support (using donations) development.
 - ✅ No, I don’t have the time and I’m okay to wait for the community / maintainers to resolve this issue.

**Comment by iamchathu** [comment]
Agreed the documentation is not updated with new migrations updates too.

**Comment by DrewImm** [comment]
Duplicates Issue #9085 (fixed in pending PR #9091)

**Comment by guisartori** [comment]
I solved the problema like this in my `package.json`:
`
"scripts": {
    ...
    "typeorm:mc": "cd src/app/database/migrations && typeorm-ts-node-commonjs migration:create"
    ...
}
`

**Comment by tarunregmi** [comment]
> Duplicates Issue [#9085](https://github.com/typeorm/typeorm/issues/9085) (fixed in pending PR [#9091](https://github.com/typeorm/typeorm/pull/9091))

but real issue still exist!

**Comment by tarunregmi** [comment]
> I solved the problem like this in my `package.json`:
> 
> ```
> "scripts": {
>     ...
>     "typeorm:mc": "cd src/app/database/migrations && typeorm-ts-node-commonjs migration:create"
>     ...
> }
> ```

This might work in a toy setup, but in real projects that rely on environment variables or path-based config, changing the `cwd` like this breaks things — because `.` now resolves relative to `src/app/database/migrations` instead of the project root. It’s a fragile hack, not a real solution.

**Comment by tarunregmi** [comment]
> I solved the problem like this in my `package.json`:
> 
> ```
> "scripts": {
>     ...
>     "typeorm:mc": "cd src/app/database/migrations && typeorm-ts-node-commonjs migration:create"
>     ...
> }
> ```

Works fine for a toy setup, but the real pain starts in an actual project — because this approach changes `process.cwd` from `.` to `./path/to/your/migration-dir`. Suddenly your environment variables, path resolutions, and config files start misbehaving. Try running `migration:generate`, `migration:run`, or `migration:revert` in this setup — it's a great way to break things you didn’t know were breakable.
